pkg/util/validation/limits.go

```diff
@@ -210,6 +213,8 @@ type Limits struct {
 	AlertmanagerMaxAlertsCount                 int                `yaml:"alertmanager_max_alerts_count" json:"alertmanager_max_alerts_count"`
 	AlertmanagerMaxAlertsSizeBytes             int                `yaml:"alertmanager_max_alerts_size_bytes" json:"alertmanager_max_alerts_size_bytes"`
 	DisabledRuleGroups                         DisabledRuleGroups `yaml:"disabled_rule_groups" json:"disabled_rule_groups" doc:"nocli|description=list of rule groups to disable"`
+
+	ExternalLabels labels.Labels `yaml:"external_labels" json:"external_labels" doc:"nocli|description=external labels for alerting rules"`
```

Comment:
Should we be more specific about the field name? Maybe something like `RuleExternalLabels` is better.
It is fine to just call it `ExternalLabels` under Ruler config but now it is under Limits so better to be more specific

Reply:
I was thinking about adding them into remote read as well, however, a separate one sounds good to me too.